modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/js/components/global-js/AttributeFields.tsx

```diff
@@ -78,12 +78,13 @@ export default function AttributeFields({
 					<ClayInput
 						aria-describedby={`${nameId}fieldFeedback`}
 						aria-required={true}
+						data-testid="name"
 						defaultValue={name}
 						disabled={disabled}
 						id={nameId}
 						onChange={(event) => {
 							const value = event.target.value
-								.split(' ')
+								.split(/[\s\u00A0]+/)
```

Comment:
Hey @ilzamcmed the regex whitespace character should handle all types of whitespace, so you shouldn't need `\u00A0`. You can just have `.split(/\s/)`.